### DISCUSSION TITLE
azure sql database

### DISCUSSION THREAD

**Discussion by sreediff** [olahallengren/sql-server-maintenance-solution#633] [General]
is the solution designed to be implemented for azure sql database (with elastic pools)

**Comment by jacobgexigo** [comment]
Yes, we use it daily...but only for specific parts of the maintenance.  IndexOptimize & DatabaseIntegrityCheck will work, but DatabaseBackup will not.  Just download and run the following scripts:

- CommandLog.sql
- Queue.sql
- QueueDatabase.sql
- CommandExecute.sql
- DatabaseIntegrityCheck.sql
- IndexOptimize.sql
